estate_account/models/estate_property.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models, Command
+
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    def set_to_sold(self):
+        for record in self:
```

Comment:
May be a better idea to call the super first, because there we check if we can actually sell the property and then create the invoice

```suggestion
    def set_to_sold(self):
        super().set_to_sold()
        for record in self:
```

Reply:
Question: You suggest to call `super` first just for clarity right? but the whole function is fully transactional and the invoice created will be rolled back if the call to `super` failed?